src/Microsoft.AspNetCore.Server.Kestrel/Internal/Http/Frame.cs

```diff
@@ -312,6 +311,7 @@ public void Reset()
         /// </summary>
         public void Start()
         {
+            Reset();
```

Comment:
Why did you move this here? It was bad in the ctor so I like that you moved it here, but just wondering if there were other reasons.